tests/unit/command_positive.cpp

```diff
@@ -612,6 +612,14 @@ TEST_F(PositiveCommand, ClearRectWith2DArray) {
     if (!AreRequiredExtensionsEnabled()) {
         GTEST_SKIP() << RequiredExtensionsNotSupported() << " not supported";
     }
+    
+    if (InstanceExtensionSupported(VK_KHR_PORTABILITY_ENUMERATION_EXTENSION_NAME)) {
+        auto portability_features = LvlInitStruct<VkPhysicalDevicePortabilitySubsetFeaturesKHR>();
+        GetPhysicalDeviceFeatures2(portability_features);
+        
+        if(portability_features.imageView2DOn3DImage == VK_FALSE)
+            GTEST_SKIP() << "Required feature 'imageView2DOn3DImage' not supported by portability extension.";
+        }
```

Comment:
Fix brackets. I know this code technically works but I had to do a double take because the formatting is awkward.

```suggestion
        if(portability_features.imageView2DOn3DImage == VK_FALSE) {
            GTEST_SKIP() << "Required feature 'imageView2DOn3DImage' not supported by portability extension.";
        }
    }
```

Reply:
Done.